test_runner/regress/test_storage_controller.py

```diff
@@ -55,9 +57,10 @@ def get_node_shard_counts(env: NeonEnv, tenant_ids):
     return counts
 
 
-def test_storage_controller_smoke(
-    neon_env_builder: NeonEnvBuilder,
-):
+@pytest.mark.parametrize(
+    "combination", [0x1F, 0x15, 0x02, 0x0C, 0x09, 0x1A], ids=lambda x: f"combination{x:05b}"
```

Comment:
It seems the combination with hexes looks pretty unreadable 🤔 
I'd create a separate function that would return this list and then add a comment with an explanation of the function.

Or even better, create a function that we'll call inside `@pytest.mark.parametrize`, something like:

```python
@pytest.mark.parametrize(some_function_name())
def test_smth(...):
    ...
```

Reply:
done